flake-module.nix

```diff
@@ -87,6 +87,22 @@ in
                 '';
                 default = false;
               };
+              hlintCheck = mkOption {
+                type = types.submodule {
+                  options = {
+                    enable = mkOption {
+                      type = types.bool;
+                      description = "Check whether hlint is enabled";
+                      default = false;  
+                    };
+                    dirs = mkOption {
+                      type = types.listOf types.str;
+                      description = "Directories that should be checked with hlint";
+                      default = [];
```

Comment:
```suggestion
                      default = ["."];
```

Reply:
A more sensible default would be `src`, which is the root of Srid's [haskell-template](https://github.com/srid/haskell-template) repo; this is the reason that we implemented the whole `find ...` command on line 186.

Do you have a better suggestion? Current directory doesn't make sense b/c you might have random Nix files and stuff that don't apply to `hlint`.

Reply:
I didn't realize that `hlint .` worked. Yea, that would be a better default.

> Current directory doesn't make sense b/c you might have random Nix files and stuff that don't apply to hlint.

hlint will only look at `.hs` files, I assume.

Reply:
If that's the case, then I have no issue